src/main/java/cz/muni/fi/pa165/entity/User.java

```diff
@@ -1,22 +1,133 @@
 package cz.muni.fi.pa165.entity;
 
 /**
- * Hello world!
+ * Robo
  *
  */
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Column;
```

Comment:
Importuj "javax.persistence.*", neni tam skoro zadny performance impact a dela to cistsi kod